source/gamemap/GameMap.cpp

```diff
@@ -2021,25 +2021,24 @@ bool GameMap::seatIsAWinner(Seat *s) const
     return false;
 }
 
-void GameMap::addGoalForAllSeats(Goal *g)
+void GameMap::addGoalForAllSeats(std::unique_ptr<Goal>&& g)
```

Comment:
you like unique_ptr as always.
I have to admit you are slowly converting me to using them ^^